nuxt.config.js

```diff
@@ -155,14 +155,6 @@ export default {
   // Nuxt Image
   image: {
     dir: 'assets/images',
-    providers: {
-      ipx: {
-        provider: 'ipx',
-        options: {
-          baseURL: 'https://formester-strapi.s3.ap-south-1.amazonaws.com/'
-        }
-      }
-    },
```

Comment:
why was this removed?

Reply:
This was causing the local images to not load.

Reply:
Just adding the domain works fine. This images are now visible in the preview.